controllers/API/announcement.js

```diff
@@ -51,7 +51,7 @@ router.get("/API/announcement", function (req, res) {
 	Announcement
 		.find(req.query)
 		.sort({ $natural: -1 })
-		.limit(req.query.count || 200)
+		.limit(parseInt(req.query.count) || 200)
```

Comment:
I find it dubious that this threw an error. Mongoose should be able to parse all strings to integers. This isn't throwing any errors on my local installation either.

Reply:
I found an issue in github that says it cant parse integers in limit. After i changed it to this it stopped throwing errors.

Reply:
Peculiar. But if it works, then sure.

Reply:
Just tested it without `parseInt` does not work on my local application or the server. Am keeping this change.